v2/api/keyvault/customizations/vault_extensions.go

```diff
@@ -19,12 +21,13 @@ import (
 
 	keyvault "github.com/Azure/azure-service-operator/v2/api/keyvault/v1api20230701/storage"
 	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1api20200601/storage"
-
```

Comment:
nit: I like separating named imports from regular imports.

Reply:
I brought these back but AFAIK that's not a common pattern so tools (gofump/goimports/goland import ordering) don't have an option to do this. They preserve them if there but AFAIK the standard ordering is either stdlib / everything else (2 groups) or stdlib / external / internal (3 groups) and this results in 4.